cypress/integration/26-register_publication/277-creator_navigates_to_summary_tab/summary.js

```diff
@@ -0,0 +1,104 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_WITH_AUTHOR } from '../../../support/constants';
+import 'cypress-localstorage-commands';
+import 'cypress-file-upload';
+
+const testFile = 'example.txt';
+
+// common steps

```

Comment:
hva betyr dette?

Reply:
Hvis scenarioer har like steps vil cypress kjøre det første steppet det finner. `common steps` er de som er felles for begge scenarioene i featuren her.

Reply:
Ok, skjønner. Kanskje lurt med en litt mer beskrivende kommentar?

Reply:
legger du på kommentarer her også?